Operators.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+
+namespace HelloWorld
+{
+    public class Operators
+    {
+        public void MathOperations()
+        {
+            int a, b, c;
+            a = 5; b = 2;
+            c = a + b;
+            b += 3;
+            a -= 4;
+
+            Debug.WriteLine($"a={a}, b={b}, c={c}");
+        }
+
+    }
+    public class NullValidation
+    {
+        public object RetrieveRandomItem()
+        {
+            List<object> myList = new List<object>() { 1, 1.1111, "Sharad", null, 'c', 2.79769313486232E+3 };
+
+            Random rnd = new Random();
+            object result = myList[rnd.Next(0, myList.Count)];
+            if (result != null) 
+                Debug.WriteLine($"random item: {result!}");
```

Comment:
it makes no sense to use ! postfix here as you've already checked if the value is null prior to this